package.json

```diff
@@ -91,17 +91,17 @@
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-tsdoc": "^0.4.0",
     "eslint-plugin-unused-imports": "^4.1.4",
-    "express": "^4.21.2",
+    "express": "^5.1.0",
     "gcp-metadata": "^5.3.0",
     "js-yaml": "^4.1.0",
-    "mocha": "^10.8.2",
+    "mocha": "^11.1.0",
     "mocha-sinon": "^2.1.2",
     "mongodb-client-encryption": "^6.3.0",
     "mongodb-legacy": "^6.1.3",
-    "nyc": "^15.1.0",
-    "prettier": "^3.4.2",
+    "nyc": "^17.1.0",
+    "prettier": "^3.5.3",
```

Comment:
The PR says "non-lint" what made this one stay? (fine with leaving it if it isn't an issue, just want to understand)

Reply:
prettier isn't a linter, its a formatter.  pedantic I guess, but that's how I broke it up in my head